plugin-tests/tests/bootstrap.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * PHPUnit bootstrap file
+ *
+ * @package performance-lab
+ */
+
+define( 'TESTS_PLUGIN_DIR', dirname( __DIR__ ) );
+
+// Determine correct location for plugins directory to use.
+if ( false !== getenv( 'WP_PLUGIN_DIR' ) ) {
+	define( 'WP_PLUGIN_DIR', getenv( 'WP_PLUGIN_DIR' ) );
+} else {
+	define( 'WP_PLUGIN_DIR', dirname( TESTS_PLUGIN_DIR ) );
+}
+
+// Load Composer dependencies if applicable.
+if ( file_exists( TESTS_PLUGIN_DIR . '/vendor/autoload.php' ) ) {
+	require_once TESTS_PLUGIN_DIR . '/vendor/autoload.php';
+}
```

Comment:
This is likely the reason that you have needed to call composer install in each plug-in directory. If this were looking for the auto load files somewhere else, everything probably would work. Not a blocker though, for my point of you.

Reply:
@joemcgill  as discussed between us, I'm having a hard time getting it to work without plugin specific installs. For now, I am going to proceed with plugin specific composer installs, as it's working. If others want to open up a PR for this after the fact, this is most welcomed.